lib/bgs/services/intent_to_file.rb

```diff
@@ -95,5 +97,12 @@ def required_update_intent_to_file_fields
         submitter_application_icn_type_code
       ]
     end
+
+    def missing_claimant_attributes!(options:)
```

Comment:
nitpick/renaming suggestion: `validate_claimant_attributes` reads more like a check, whereas `missing_claimant_attributes` just looks like an error the way it's called.